### PR TITLE
WIP - CVE-2015-3456 - venom

### PR DESCRIPTION
**Pull request by paulczar**

### CODE REVIEW

File: playbooks/CVE-2015-3456.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  tasks:
+  - apt: update_cache=yes cache_valid_time=3600
+  - name: Patch CVE-2015-3456
+    apt: pkg={{ item }} state=latest
+    with_items:
+    - qemu
+    - qemu-kvm
+    notify: restart qemu-kvm
+  handlers:
+  - name: restart qemu-kvm
```
Comment (dlundquist) [inline playbooks/CVE-2015-3456.yml]: is this actually a service, or individual processes spawned by libvirt?

File: playbooks/CVE-2015-3456.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+- hosts: all
```
Comment (dlundquist) [inline playbooks/CVE-2015-3456.yml]: We should only target the compute role.

File: playbooks/CVE-2015-3456.yml (hunk 1 above)
Comment (paulczar) [inline playbooks/CVE-2015-3456.yml]: this is actually a service.

```
sudo restart qemu-kvm
qemu-kvm start/running
```

File: bin/run_vagrant [hunk 3]
```diff
@@ -49,6 +49,7 @@ done
 
 export ANSIBLE_SSH_ARGS="${ANSIBLE_SSH_ARGS} -F ${TEMPFILE}"
 command="ursula envs/vagrant/${TYPE} site.yml -s -u vagrant $@"
+command="ursula envs/vagrant/${TYPE} playbooks/CVE-2015-3456.yml -s -u vagrant $@"
```
Comment (dlundquist) [inline bin/run_vagrant]: Did you intend to commit this?

**Comment by dlundquist** [comment]
Looks like this still needs an updated QEMU version for Precise.

File: bin/run_vagrant (hunk 3 above)
Comment (paulczar) [inline bin/run_vagrant]: that's just there to make it easier for testing,  will remove it once this is no longer WIP.

**Comment by paulczar** [comment]
Correct.  we're waiting on the cloud archive to get a patched version of qemu.  otherwise we'd have to revert back to qemu 1.x

**Comment by dlundquist** [comment]
Looking at https://launchpad.net/ubuntu/+source/qemu/+changelog, it should be `qemu-kvm=2.0.0+dfsg-2ubuntu1.11~cloud0`.

**Pull request closed without merging**